add support for card taps

fouadkada opened the pull request:
This library seems to be the cleanest out there but for some reason a listener for card taps is omitted.
this PR add a callback for card taps

fouadkada committed:
add support for card taps

diogobernardino commented:
Sounds good the idea.

I would implement `GestureDetector.OnDoubleTapListener` instead of extending `GestureDetector.SimpleOnGestureListener`.

`FlingChief `object should have no clue about a deck/card existence, it should be fully independent, so I would stick to this idea and change the callback name. Maybe just `onTapped()`?

fouadkada committed:
rename method and introduce listener for double taps

fouadkada commented:
@diogobernardino i renamed the method as suggested and introduced a listener for double taps.
I believe some users will be interested in the single tap event and others will be interested in the double tap event
